.eslintrc

```
@@ -0,0 +1,9 @@
+{
+  "extends": "standard",
```

Just tried testing pre-commit hook with a small change in **luigi-client-link-manager-features.spec.js** .
It seems like ESLint doesn't recognize cypress properly (cy is undefined). I guess it would be good to at least have support for cypress in the linter since we use it a lot for testing.
Referring from this issue
```suggestion
  "extends": ["standard", "plugin:cypress/recommended"],
```

also add the dependency in the other place.